.github/workflows/sonarqube-scan.yml

```diff
@@ -0,0 +1,99 @@
+name: SonarQube Code Scan
+on:
+  workflow_dispatch:
+  workflow_call:
+
+env:
+  SONAR_TOKEN: sqp_db041dcd2f5c1f5310d1a87bde0dde3c52b455fa
```

Comment:
This smells like a secret to me.  Now that it is exposed, you might want to regenerate a new token. Put this in [GHA secrets](https://github.com/ucsf-education/moodle/settings/secrets/actions)

Reply:
Official comment: this usage is approved in consultation with the UCSF SQ owner. There is no more effective way to use this service with forks because SQ doesn't support `pull_request_target` or a better secret system.